src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.macOS.cs

```diff
@@ -128,7 +128,7 @@ protected override Size MeasureOverride(Size size)
 				{
 					// This measures the height correctly, even if the Text is null or empty
 					// This matches Windows where empty TextBlocks still have a height (especially useful when measuring ListView items with no DataContext)
-					var font = NSFontHelper.TryGetFont((float)FontSize, FontWeight, FontStyle, FontFamily);
+					var font = NSFontHelper.TryGetFont((float)FontSize*2, FontWeight, FontStyle, FontFamily);
```

Comment:
Hmm... I get why, but is this the good fix?

Either way, needs a bit more green.

Reply:
No green for now. It'll be addressed in another PR.